pipelines/py/pip-check.yaml

```diff
@@ -0,0 +1,42 @@
+name: Use pip from a zipapp for bootstrap. Only for use with bootstrapping.
```

Comment:
please make the name and comments meaningful for what this pipeline does.

Reply:
done.